src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -9,6 +9,7 @@
 import edu.wpi.first.wpilibj.drive.DifferentialDrive;
 import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
```

Comment:
Notice that this imports `Constants.*`, not `DriveTrainConstants.*` which is the one you want.
`DriveTrainConstants` is imported a line below.